tests/run-locally.sh

```diff
@@ -9,12 +9,7 @@ trap finish EXIT
 
 docker run --detach --publish 5000:5000 --rm --name registry registry:2
 
-docker pull busybox
-docker tag busybox libero/my-dummy-project:12345678
-
-GITHUB_REF=refs/heads/master \
 DOCKER_REGISTRY=localhost:5000/ \
 DOCKER_USERNAME=foo \
 DOCKER_PASSWORD=bar \
-IMAGE_TAG=12345678 \
-github/push-image.sh my-dummy-project
+bats -r tests/
```

Comment:
This *must* be run in the root of the project (maybe we can add a check?)

Reply:
I think this would be best addressed by adding a makefile, no?